db/migrate/20180920201052_create_versions.rb

```diff
@@ -0,0 +1,80 @@
+# This migration creates the `versions` table, the only schema PT requires.
+# All other migrations PT provides are optional.
+class CreateVersions < ActiveRecord::Migration
+  # Class names of MySQL adapters.
+  # - `MysqlAdapter` - Used by gems: `mysql`, `activerecord-jdbcmysql-adapter`.
+  # - `Mysql2Adapter` - Used by `mysql2` gem.
+  MYSQL_ADAPTERS = [
```

Comment:
Looks like it has a `mysql?` method to add fields specific to MYSQL.  Example: 
`opt[:limit] = 191 if mysql?`

This would only be useful if we change the DB in the future, and only if it runs all the migrations rather than loading schema.  I'd say we leave it in, given that it's auto-generated and isn't doing any harm.  It might be helpful for people in the future.